parser/src/lexer.rs

```diff
@@ -99,6 +99,7 @@ pub fn get_keywords() -> HashMap<String, Tok> {
 
     // Alphabetical keywords:
     keywords.insert(String::from("..."), Tok::Ellipsis);
+    keywords.insert(String::from("Ellipsis"), Tok::Ellipsis);
```

Comment:
Ellipsis is a builtin name. See my other comment. You might want to add this testcase:
```python
assert Ellipsis is ...
Ellipsis = 2
assert Ellipsis is not ...
```